tools/c7n_org/c7n_org/cli.py

```diff
@@ -404,11 +404,13 @@ def run_script(config, output_dir, accounts, tags, region, echo, serial, script_
 
 
 def accounts_iterator(config):
-    for a in config.get('accounts'):
-        yield a
-    for a in config.get('subscriptions'):
-        d = {'account_id': a['subscription_id'], 'name': a['name']}
-        yield d
+    if config.get('accounts'):
```

Comment:
rather than if, do the loop with for a in config.get('accounts', ())

Reply:
fwiw, there's a followup branch on gcp org support that has these changes as well.